internal/configconverter/otlp_histogram_attr.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configconverter
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"go.opentelemetry.io/collector/confmap"
+)
+
+type AddOTLPHistogramAttr struct{}
+
+// Convert updates the service::telemetry::resource to add the attribute splunk.otlp.histograms=true.
+// This additional resource attr is only added if we see any signalfx exporter in use with the config
+// send_otlp_histograms set to true.
+func (AddOTLPHistogramAttr) Convert(_ context.Context, cfgMap *confmap.Conf) error {
+	if cfgMap == nil {
+		return nil
+	}
+
+	sfxExporters := map[string]bool{}
```

Comment:
It won't make a real difference, but, since this is used only as a presence test I think that a `map[string]struct{}` will read better since it makes clear that nothing will be stored for each key.